host/host_darwin_amd64.go

```diff
@@ -1,5 +1,6 @@
 // Created by cgo -godefs - DO NOT EDIT
 // cgo -godefs types_darwin.go
+// +build go1.6,amd64
```

Comment:
I can not confirm right now, however, from the git log, this file is introduced  at `Nov 27 10:18:15 2014`, go 1.4 age. It could be compiled at that time.